tests/e2e/performance/test_pvc_creation_deletion_performance.py

```diff
@@ -165,7 +218,10 @@ def test_pvc_creation_deletion_measurement_performance(
             logging.info(f"{msg_prefix} Start creating PVC number {i + 1}.")
             start_time = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
             pvc_obj = helpers.create_pvc(sc_name=self.sc_obj.name, size=pvc_size)
-            helpers.wait_for_resource_state(pvc_obj, constants.STATUS_BOUND)
+            timeout = 3600 if self.interface == constants.CEPHBLOCKPOOL_THICK else 60
```

Comment:
attaching an rbd thick provisioned PVC should take the same time as rbd PVC as the initialization is already done during the creation of the PVC.

Reply:
@keesturam I'll check whether the test will pass with timeout =6- for reaching status Bound. 
If not - should I open a bug ?

Reply:
@keesturam  60 secs is not enough - all the RBD thick PVCs do not reach bound state within this time. Is it a bug? 
I'm going to test timeout = 600 now. 
Are you sure that time for reaching bound state is not part of the creation time ?